docs/buildstrategies.md

```diff
@@ -187,6 +187,16 @@ kubectl apply -f samples/buildstrategy/source-to-image/buildstrategy_source-to-i
 [s2i]: https://github.com/openshift/source-to-image
 [buildah]: https://github.com/containers/buildah
 
+## System parameters
+
+You can use parameters when defining the steps of a build strategy to access system information as well as information provided by the user in his Build or BuildRun. The following parameters are available:
+
+| Parameter                      | Description |
+| ------------------------------ | ----------- |
+| `$(params.shp-source-root)`    | The absolute path to the directory that contains the user's sources. |
+| `$(params.shp-source-context)` | The absolute path to the context directory of the user's sources. If the user specified no value for `spec.source.contextDir` in his Build, then this value will equal the value for `$(params.shp-source-root)`. Note that this directory is not guaranteed to exist at the time the container for your step is started, you can therefore not use this parameter as a step's working directory. |
+| `$(params.shp-image-url)`      | The URL of the image that the user wants to push as specified in the Build's `spec.output.image`, or the override from the BuildRun's `spec.output.image`. |
```

Comment:
Since this is related to output, should we consider `shp-output-image` as the parameter name? This will communicate:

1. That this parameter is for the output image
2. That it is an image reference

We should make some guarantees with respect to this value - for example, that will always be a valid image reference. An image ref is not necessarily a URL - for example, `busybox:latest` is a perfectly valid reference.

In the future we may want to support OCI images as input sources, therefore "image-url" may not be sufficiently specific.

Reply:
Done.